lib/cfg.js

```diff
@@ -13,7 +13,13 @@ function resolvePath(unresolvedPath) {
 module.exports = function(main, opts) {
   var dir = main ? path.dirname(main) : '.'
   var c = read(dir)
-
+  if(c==null && dir!='.'){
+    dir = path.dirname('.');
+    c = read(dir);
+  }
+  if(c==null){
+    c={}
+  }
```

Comment:
Minor style: Please add spaces around operators such as `==` and `!=` and use their strict equivalents, i.e. `===` and `!==`. Please omit semicolons like the rest of the codebase. Hopefully we can put in a lint tool soon :+1: